src/org/opensolaris/opengrok/web/PageConfig.java

```diff
@@ -752,6 +752,20 @@ public String getRequestedProjectsAsString() {
         return requestedProjects;
     }
     
+	public SortedSet<String> getAllRequestedProjects() {
+		    TreeSet<String> set = new TreeSet<>();
+            Set<Project> allProjects = getProjectHelper().getAllProjects();
```

Comment:
I thought there is a need to use authFramework.isAllowed to check if a search is allowed?